packages/editor-ui/src/constants.ts

```diff
@@ -144,6 +144,12 @@ export const OPEN_AI_NODE_MESSAGE_ASSISTANT_TYPE =
 export const OPEN_AI_ASSISTANT_NODE_TYPE = '@n8n/n8n-nodes-langchain.openAiAssistant';
 export const BASIC_CHAIN_NODE_TYPE = '@n8n/n8n-nodes-langchain.chainLlm';
 export const QA_CHAIN_NODE_TYPE = '@n8n/n8n-nodes-langchain.chainRetrievalQa';
+export const MEMORY_BUFFER_WINDOW_NODE_TYPE = '@n8n/n8n-nodes-langchain.memoryBufferWindow';
```

Comment:
There might be a good reason why these were not yet defined? Do we usually exclude subnodes?

Reply:
No no, we don't. This wasn't defined because we simply didn't need it yet in editor-ui :D